transformer-intro/README.md

```diff
@@ -0,0 +1,389 @@
+# Basic Transformer
+
+In this example we will use Marian to create a English-German translation
```

Comment:
Consider mentioning that this is a basic tutorial that should be a good fit for new practitioners into Marian and machine translation in general.